src/app/components/ShepContainer/ShepContainer.jsx

```diff
@@ -1,53 +1,80 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
+import Store from '@Store';
 import Breadcrumbs from '../Breadcrumbs/Breadcrumbs';
 import LoadingLayer from '../LoadingLayer/LoadingLayer';
 
-import Store from '../../stores/Store';
+class ShepContainer extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isLoading: Store.state.isLoading,
+    };
+    this.onChange = this.onChange.bind(this);
+    Store.listen(this.onChange);
```

Comment:
Start implementing redux

Reply:
Can this be done without listening to the Store here?

Reply:
I think you also need to "unlisten" to the Store when this is unmounted.